_images/project-data/migrating-legacy-designer2-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='migrating-from-legacy-designer-proc-legacy-designer-proc']
+
+= Migrating projects from the deprecated legacy process designer
+
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users of the legacy process designer should start migrating projects to the new designer and all new projects should be created using the new process designer.
```

Comment:
I believe that in this paragraph instead of saying that:

"Existing users of the legacy process designer should start migrating projects to the new designer and all new projects should be created using the new process designer."

we could say that they "should start migrating their processes to the new designer and all new processes should be created using the new process designer"

Since the "project" itself is not migrated because of the new designer, instead the "processes" should be migrated.

But also it might also be worth to say at some point that the jBPM execution engine still support to execute processes generated with the legacy jBPM designer. So old projects that contain processes generated with the legacy designer, will still be executable/deployable into kie-server. It could be the case that a user has a project with processes generated with old jbpm designer, and he's happy with this processes, and this processes will remain as is with no changes, so in this cases it's not mandatory for the user to start migrating this processes. 
Not sure what is the message we want to give to the user here, if the message should be to "start migrating all", or to "start migrating what you really need". (if I'm have a project that I'm happy with, and this project won't change, do I have to migrate it?)  
I'd suggest asking @krisv and @ibek to clarify the message to give

Reply:
Good point. I have added the following statement:

The jBPM execution engine will continue to support the execution and deployment of business processes generated with the legacy designer in to the {SERVER}. If you have a legacy business process that is functioning and that you do not intend to change, it is not mandatory to migrate to the new designer at this time.